app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -58,42 +58,61 @@ public static void setImageDrawable(
     switch (thumbnailGraphic) {
       case BAKER:
         drawableResourceId = R.drawable.lesson_thumbnail_graphic_baker;
+        break;
```

Comment:
Oh yikes, how was this working so far !

Reply:
This is the expected behavior right? There should be no fallthrough?

Reply:
mentioning @rt4914 to confirm this is the expected behaviour.

Reply:
@vinitamurthi @anandwana001 
`break` should be there and even I am wondering how is this working correctly in develop.

@anandwana001 Please add break after all these cases in switch.

Reply:
![Screenshot_1605154019](https://user-images.githubusercontent.com/9396084/98894486-dde99a80-24ca-11eb-8910-43321062ca31.png)

After having discussing with @anandwana001 we found that it is not working in walkthrough and is getting used only there.

Reply:
I'm guessing that this was missed when the file was translated to Java. The original was probably something like:

```kotlin
return when (state) {
  VALUE1 -> result1
  VALUE2 -> result 2
  ...
}
```

I'm glad this was found.

Reply:
Also, @anandwana001 can you file an issue to track improving the code coverage here? We should have caught this in tests, but we apparently aren't covering these lines correctly.

Reply:
Done - https://github.com/oppia/oppia-android/issues/2449